tools/regionalGAM/scriptsR_Galaxy/flight_curve_en.xml

```diff
@@ -4,13 +4,12 @@
         <requirement type="package" version="3.4">r</requirement>
```

Comment:
We can change this 
`<requirement type="package" version="3.4">r</requirement>`
by this :
`<requirement type="package" version="1.12.2">r-data.table</requirement>`

Reply:
Thank you for the comment, for this line, why do I have to replace the line and not just add it under ?
like this:
```
<requirement type="package" version="3.4">r</requirement>
<requirement type="package" version="1.12.2">r-data.table</requirement>
```

Reply:
because r-data.table has its own requirements and will automatically install the r version that better suits him.

Reply:
You can still specify a r version if needed, and both requirement will be applied if compatible.

Reply:
That is correct, the best was is to use r-data.table as @Alanamosse pointed out. If you really want to include R, the better package to use is `r-base` and probably in version 3.5.

Reply:
Thank you both for this explanation !